spec/rails_admin/support/csv_converter_spec.rb

```diff
@@ -151,5 +151,32 @@
         expect(subject[2]).to eq("\n")
       end
     end
+
+    context 'when more than one page of objects' do
+      before do
+        FactoryBot.create_list :player, 100
+      end
+
+      let(:objects) { Player.all }
+      let(:options) { {} }
+
+      it 'generates a csv with all objects' do
+        expect(subject[2].split("\n").count).to be 101
+      end
+    end
+
+    context 'when objects are ordered' do
```

Comment:
If this is okay to you, please also consider matching whole records. This will make sure of order even for other code changes.
```rb
context 'when objects are ordered' do
  # or use before block as before
  let!(players) do
    FactoryBot.create_list :player, 2 do |player, index|
      player.name = "Player #{index}"
    end
  end

  let(:objects) { Player.all.order(name: :desc) }
  let(:options) { {} }

  # modify the subject block to return the array of objects
  it 'preserves the ordering' do
    expect(subject[2].split("\n")[1]).to eq('Player 1', 'Player 2')
  end
end
```

Reply:
Thanks. I incorporated some of your suggestions, but matching the whole record is a bit nontrivial because the pk ids are not predictable when randomizing test seed